pom.xml

```diff
@@ -165,6 +165,16 @@
                 <artifactId>json-schema-validator</artifactId>
                 <version>2.2.10</version>
             </dependency>
+            <dependency>
+                <groupId>com.github.workerframework.testing</groupId>
+                <artifactId>workerframework-testing-util</artifactId>
+                <version>1.0.0-SCMOD-5524-SNAPSHOT</version>
+            </dependency>
+            <dependency>
+                <groupId>com.github.workerframework.testing</groupId>
+                <artifactId>workerframework-testing-integration</artifactId>
+                <version>1.0.0-SCMOD-5524-SNAPSHOT</version>
```

Comment:
Don't forget to update the two version numbers before merging.

Reply:
@dermot-hardy  for all downstreams using that dependency "https://github.com/CAFDataProcessing/worker-document/tree/develop/worker-document-testing". Should we simply update the dependency version, or should we also explicitly declare the [com.github.workerframework.testing](https://github.com/WorkerFramework/worker-framework-testing) (new extracted project) dependencies

Reply:
You can declare them in dependency management.  Don't list them as dependencies unless they are actually direct dependencies.